apps/hash-frontend/tsconfig.json

```diff
@@ -11,7 +11,12 @@
       "@blockprotocol/type-system": [
         "../../../../node_modules/@blockprotocol/type-system/dist/cjs-slim/index-slim.d.ts"
       ]
-    }
+    },
+    "plugins": [
+      {
+        "name": "next"
```

Comment:
This might only offer benefits for checking App router usage but isn't seeming to do any harm here at least, and they might add useful rules for non-App router stuff later.